qiita_pet/templates/list_studies.html

```diff
@@ -11,6 +11,11 @@
 
 <script type="text/javascript">
 var current_study;
+var ws = new WebSocket('ws://' + window.location.host + '/analysis/selected/socket/');
```

Comment:
same here on construction. if not moi's init, the construction needs to be centralized somehow otherwise boilerplate like browser checks will become tedious